Jenkinsfile

```diff
@@ -61,7 +61,13 @@ pipeline {
         stage('Parallel'){
           when {
             beforeAgent true
-            expression { return env.ONLY_DOCS == "false" }
+            allOf {
+              expression { return env.ONLY_DOCS == "false" }
+              anyOf {
+                changeRequest()
+                branch '**/*'
```

Comment:
I could not find any other way to filter but using the `glob` 

<img width="655" alt="image" src="https://user-images.githubusercontent.com/2871786/157733381-15a32d0f-70b2-4d0d-8aab-2da041daf055.png">